test/fakeintake/server/server_test.go

```diff
@@ -42,6 +42,59 @@ func TestServer(t *testing.T) {
 		assert.Equal(t, "server not running", err.Error())
 	})
 
+	for _, tt := range []struct {
+		name string
+		port string
+		opt  Option
+	}{
+		{
+			name: "Make sure WithPort sets the port correctly",
+			port: "1234",
+			opt:  WithPort(1234),
+		},
+		{
+			name: "Make sure WithAddress sets the port correctly",
+			port: "3456",
+			opt:  WithAddress(":3456"),
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
To avoid macos screaming and popping up
```suggestion
			opt:  WithAddress("127.0.0.1:3456"),
```

Reply:
Fro address you should test the address, rather than `127.0.0.1`

Reply:
I am not sure how to make this work: `Get "http:/[::]:1234/fakeintake/health": http: no Host in request URL`

Reply:
Did it in the last commit 👍